lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
+  extend self
+  
+  def args(*currency)
+    currency = currency.flatten.compact
```

Comment:
And without `compact` call like `business_days('EUR', nil) would fail (maybe it should?)